.github/workflows/ci.yaml

```diff
@@ -0,0 +1,57 @@
+name: ci
+
+on:
+  pull_request:
+
+jobs:
+  check-for-chart-changes:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Lint Bash scripts
+        uses: docker://koalaman/shellcheck-alpine:v0.7.0
+        with:
+          args: /github/workspace/.ci/lint-scripts.sh
+      - name: Check for chart changes
+        run: /home/runner/work/helm-charts/helm-charts/.ci/check-for-chart-changes.sh
```

Comment:
this path depends on the GH repository name (`helm-charts/helm-charts`), correct?
So that should be adjusted to the "packages" repo.

Reply:
replaced with apckages/packages